### ISSUE TITLE
Option to exclude Final from reportMissingTypeArgument

### ISSUE THREAD

**Issue by twoertwein** [microsoft/pyright#2024]
**Is your feature request related to a problem? Please describe.**
`reportMissingTypeArgument` is really useful to improve type annotations, but it would be helpful to have an option to exclude `typing.Final`.

It is often clear which type a Final variable has:
```py
constant: Final = ("a", "b")
```
Encouraging more fine-grained annotations might lead to "not-ideal" annotations like
```py
constant: Final[tuple[str, str]] = ("a", "b")
```
while
```py
constant: Final[tuple[Literal["a"], Literal["b"]]] = ("a", "b")
```
would be the "ideal" (but lengthy) annotation that pyright/mypy would infer from the first example.

**Describe the solution you'd like**
An additional  configuration parameter to exclude Final from `reportMissingTypeArgument`.

**Comment by erictraut** [comment]
Yeah, I agree that `Final` shouldn't require type arguments. It's not really a generic class, after all.

I've made the change (simply involved deleting one line of code). The fix will be in the next release.

Thanks for the suggestion.

**Comment by erictraut** [comment]
This is included in pyright 1.1.152, which I just published. It will also be included in the next release of pylance.
